Error building 'demodadsb'

Using MSVC on Win-10 to build the **demodadsb** plugin, causes this error:
```c
adsbdemodsinkworker.h(54): error C2079: 'ADSBDemodSinkWorker::MsgConfigureADSBDemodSinkWorker::m_settings' uses undefined struct 'ADSBDemodSettings' 
adsbdemodsinkworker.h(79): error C2079: 'ADSBDemodSinkWorker::m_settings' uses undefined struct 'ADSBDemodSettings'
adsbdemodsinkworker.h(44): error C2440: 'return': cannot convert from 'const int' to ' const ADSBDemodSettings &' 
...
```

But with this change it now builds for me:
```diff
--- a/adsbdemodsinkworker.h 2024-11-15 09:16:23
+++ b/adsbdemodsinkworker.h 2024-11-15 10:39:28
@@ -26,6 +26,7 @@
 #include "util/message.h"
 #include "util/messagequeue.h"
 #include "adsbdemodstats.h"
+#include "adsbdemodsettings.h"

 class ADSBDemodSink;
-struct ADSBDemodSettings;
-struct ADSBDemodStats;
```

--- a/plugins/channelrx/demodadsb/adsbdemodsinkworker.cpp
+++ b/plugins/channelrx/demodadsb/adsbdemodsinkworker.cpp
@@ -25,7 +25,6 @@
 #include "adsbdemodreport.h"
 #include "adsbdemodsink.h"
 #include "adsbdemodsinkworker.h"
-#include "adsbdemodsettings.h"
 #include "adsb.h"
 
 MESSAGE_CLASS_DEFINITION(ADSBDemodSinkWorker::MsgConfigureADSBDemodSinkWorker, Message)

--- a/plugins/channelrx/demodadsb/adsbdemodsinkworker.h
+++ b/plugins/channelrx/demodadsb/adsbdemodsinkworker.h
@@ -26,10 +26,9 @@
 #include "util/message.h"
 #include "util/messagequeue.h"
 #include "adsbdemodstats.h"
+#include "adsbdemodsettings.h"
 
 class ADSBDemodSink;
-struct ADSBDemodSettings;
-struct ADSBDemodStats;
 
 class ADSBDemodSinkWorker : public QThread {
     Q_OBJECT

